modules/create-crio-default-capabilities.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-crio-default-capabilities_{context}"]
+= Creating a drop-in file for CRI-O's default capabilities
```

Comment:
```suggestion
= Creating a drop-in file for the default capabilities of CRI-O
```
Per IBM Style, avoid using the possessive with abbreviations or product names: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives